guides/source/installing_ruby.md

```diff
@@ -0,0 +1,105 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Installing Ruby Guide
+=====================
+
+This guide will walk you through installing the Ruby programming language on your operating system.
+
+While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
```

Comment:
Without wanting to stray too far from an opinionated "if you're sufficiently unsure that you're asking how to install Ruby, just do this", is it maybe worth still including a bit of "other ways are available" padding, just to be clear that while we're not enumerating them, the various alternatives are equally valid and supported?